plugins/statistics.koplugin/main.lua

```diff
@@ -304,6 +305,21 @@ function ReaderStatistics:updateTotalStat()
     return total_stats
 end
 
+
+--https://gist.github.com/jesseadams/791673
+function ReaderStatistics:secondsToClock(seconds)
+    local seconds = tonumber(seconds)
+    if seconds == 0 or seconds ~= seconds then
```

Comment:
you should assign tonumber(seconds) to a new var name before the comparison, otherwise `seconds ~= seconds` will always be false.